### DISCUSSION TITLE
minio buckets not found

### DISCUSSION THREAD

**Discussion by emmataobao** [minio/minio#13439] [Q&A]
![1634194766(1)](https://user-images.githubusercontent.com/46233536/137267409-6f4ad20a-78b7-4707-8744-786bba4374f1.png)
![1634194800(1)](https://user-images.githubusercontent.com/46233536/137267470-6ecf26ea-093a-4cbe-8cba-bb3d31a4bb77.png)
Use an administrator account but can't find the buckets

**Comment by harshavardhana** [comment]
> ![1634194766(1)](https://user-images.githubusercontent.com/46233536/137267409-6f4ad20a-78b7-4707-8744-786bba4374f1.png) ![1634194800(1)](https://user-images.githubusercontent.com/46233536/137267470-6ecf26ea-093a-4cbe-8cba-bb3d31a4bb77.png) Use an administrator account but can't find the buckets

Please explain properly your setup. There nothing explained in the issue for us to help you.

**Reply by emmataobao** [reply]
already create  bucket  Dashboard show  5 bucket   but  Admin:buckets   not found。

**Reply by ravindk89** [reply]
- What version of MinIO are you running
- How did you deploy MinIO? What *specific* startup command + options, redacting passwords?
- What user are you trying to log in as
- What `s3:<Action>` permissions does that user have?
- Do you see this problem if you log in with the root credentials?

These are some basic questions that we need to have to do any sort of troubleshooting.
